src/levanter/infra/ray_tpu.py

```diff
@@ -162,9 +162,16 @@ class TPUHostInfo:
 
 # Timeouts (in seconds)
 _HEALTH_CHECK_TIMEOUT = 60
-_TEARDOWN_ACTOR_TIMEOUT = 300
-_TERMINATE_ACTOR_TIMEOUT = 300
-_START_ACTOR_TIMEOUT = 7 * 24 * 60 * 60  # 1 week
+_TEARDOWN_ACTOR_TIMEOUT = 60
+_TERMINATE_ACTOR_TIMEOUT = 60
+_START_ACTOR_TIMEOUT = 60
+# NOTE: It is good to have a short timeout here because
+# cluster autoscaling happens outside the this timeout.
```

Comment:
wait is that true though? Autoscaling happens in response to demand, which create_actor is the source of? (I agree the preempted loop mitigates but I don't love it being used for this since the length of time it takes to spin up a node is on the order of ~5-10 min even if capacity is there)

Reply:
I can't find documentation for this, but this is what I noticed empirically. When you spawn an actor and there isn't enough resources for it, it will block on trying to get resources from the autoscaler, and I noticed it waiting for much longer than whatever the timeout is here. I can investigate this further before merging.